sysutils/podman/Portfile

```diff
@@ -24,6 +24,13 @@ checksums           ${distname}${extract.suffix} \
                         sha256  ee3b640029a67bc2ea563bdd27946e0a39ec83322a5a9c90bf91e206ce708caf \
                         size    10917695
 
+# Podman machine is currently only supported on Linux and Intel Macs, see:
+# * https://podman.io/blogs/2021/09/06/podman-on-macs.html
+# * https://github.com/containers/podman/issues/10577
+if { !(${configure.build_arch} eq "arm64") } {
```

Comment:
perhaps `if {${configure.build_arch} ne "arm64"} {` is more readable

Reply:
Done, thanks 🌟